src/pretix/base/templatetags/html_datetime.py

```diff
@@ -0,0 +1,58 @@
+#
+# This file is part of pretix (Community Edition).
+#
+# Copyright (C) 2014-2020  Raphael Michel and contributors
+# Copyright (C) 2020-today pretix GmbH and contributors
+#
+# This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
+# Public License as published by the Free Software Foundation in version 3 of the License.
+#
+# ADDITIONAL TERMS APPLY: Pursuant to Section 7 of the GNU Affero General Public License, additional terms are
+# applicable granting you additional permissions and placing additional restrictions on your usage of this software.
+# Please refer to the pretix LICENSE file to obtain the full terms applicable to this work. If you did not receive
+# this file, see <https://pretix.eu/about/en/license>.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License along with this program.  If not, see
+# <https://www.gnu.org/licenses/>.
+#
+from datetime import datetime
+
+from django import template
+
+from pretix.helpers.templatetags.date_fast import date_fast
+
+register = template.Library()
+
+
+@register.simple_tag
+def html_datetime(value: datetime, dt_format: str = "SHORT_DATE_FORMAT", **kwargs):
+    """
+    Building a <time datetime='{html-datetime}'>{human-readable datetime}</time> html string,
+    where the html-datetime as well as the human-readable datetime can be set
+    to a value from django's formats.py/FORMAT_SETTINGS via comma-separated arguments.
+
+    If only one argument is given, it will be used as the human-readable datetime attribute,
+    and the html datetime attribute will be set to the datetime in iso-format.
+
+    Usage example:
+    {% html_datetime event_start|localtime "SHORT_DATE_FORMAT" attr_fmt="SHORT_DATE_FORMAT" as start_date %}
+    {{start_date}}
+    """
+    if value in (None, ''):
+        return ''
+    attr_fmt = kwargs["attr_fmt"] if kwargs else None
+
+    try:
+        if not attr_fmt:
+            date_html = value.isoformat()
+        else:
+            date_html = date_fast(value, attr_fmt)
+
+        date_human = date_fast(value, dt_format)
+        return f"<time datetime='{date_html}'>{date_human}</time>"
```

Comment:
See icon.py and use `from django.utils.html import format_html` to return a safe string, that is not html-escaped and does not need to be mapped to a variable and filtered through `|safe`:

```suggestion
        return format_html("<time datetime='{}'>{}</time>", date_html, date_human)
```